gamla/functional.py

```diff
@@ -523,3 +523,24 @@ def divide_by(y):
         return y / x
 
     return divide_by
+
+
+_GET_IN_EXCEPTIONS = (KeyError, IndexError, TypeError)
```

Comment:
when can there be a TypeError ?

Reply:
if you try to access a list element with a string
not sure if we care about it